src/utils/email/sendEmail.ts

```diff
@@ -0,0 +1,43 @@
+import * as env from "dotenv";
+import nodemailer from "nodemailer";
+import moment from "moment";
+import ejs from 'ejs';
+env.config();
+
+const transporter = nodemailer.createTransport({
+  service: "Yandex",
+  auth: {
+    user: "owl@rctech.club",
+    pass: process.env.NODEMAILER_PASSWORD,
+  },
+});
+
+export const sendEmail = ({ user, booking, room }) => {
+  const { email, first_name } = user;
+  const { start, end } = booking;
+  const { number, name } = room;
+
+  const room_detail = `${name} (${number})`;
+  const date = moment(start).format("Do MMM YYYY");
+  const start_time = moment(start).format("h:mm");
+  const end_time = moment(end).format("h:mm a");
+  const emailTemplateData = { first_name, room_detail, date, start_time, end_time };
+  
+  const ejsFilePath = '/../../../src/utils/email/emailTemplate.ejs';
```

Comment:
this is a weird one indeed

Reply:
One solution that might works is moving `emailTemplate.ejs` to `./dist/utils/email` folder. But I wasn't so sure to do that since the dist folder itself is production sensitive... I guess?